src/Api.php

```diff
@@ -83,6 +83,7 @@ public function createIntent(\Payment $payment) {
       return PaymentIntent::create([
         'amount'   => (int) ($one_off->totalAmount(TRUE) * 100),
         'currency' => $payment->currency_code,
+        'setup_future_usage' => $recurring->line_items ? 'off_session' : 'on_session',
```

Comment:
I think the value for non-recurring payments should be `NULL` not `on_session` (that’s for setting up future payments where the customer is present in the checkout flow, which is not something we intend to do)

Reply:
Thanks, I’ve pushed the change.